tests/integration/helpers.bash

```diff
@@ -490,3 +513,11 @@ function teardown_bundle() {
 	done
 	rm -rf "$ROOT"
 }
+
+function requires_kernel() {
+	MAJOR_REQUIRED=$(echo "$1" | cut -d. -f1)
+	MINOR_REQUIRED=$(echo "$1" | cut -d. -f2)
```

Comment:
Use `local` and lowercase for local variables.

IOW
```bash
local major_req minor_req
major_req=$(echo "$1" | cut -d. -f1)
minor_req=$(echo "$1" | cut -d. -f2)
```

Reply:
Fixed, thanks!